google/cloud/workflows/README.md

```diff
@@ -34,6 +30,7 @@ to get started using this client library in a larger project. The following
 this library.
 
 <!-- inject-quickstart-start -->
+
```

Comment:
The code here is generated by a script.  Should we fix the script too?

Reply:
Yes, eventually, I think.  See above.